src/main/java/baron/util/TextUi.java

```diff
@@ -0,0 +1,29 @@
+package baron.util;
+
+import baron.messages.Messages;
+import baron.tasks.TaskManager;
+
+public class TextUi {
```

Comment:
I like how you have abstracted out the UI from Duke main logic 😄 !

Reply:
Thank you! :)